app/server/lib/OIDCConfig.ts

```diff
@@ -314,13 +316,19 @@ export class OIDCConfig {
     });
   }
 
-  private _sendErrorPage(req: express.Request, res: express.Response, userFriendlyMessage?: string) {
+  private _sendErrorPage(
+    req: express.Request,
+    res: express.Response,
+    userFriendlyMessage?: string,
+    targetUrl: string = "/"
```

Comment:
Would it be better for this default be `undefined`? It's used in one place (when session is unavailable), and setting `errTargetUrl` in this case seems unhelpful and confusing.

Reply:
> Would it be better for this default be `undefined`? It's used in one place (when session is unavailable), and setting `errTargetUrl` in this case seems unhelpful and confusing.


@jonathanperret , just checking if you saw this question. The diff changed a bit, but I think this question still applies: in the one place where `_sendErrorPage` is called without a `targetUrl`, is still better to replace the URL with home (since that's may not be the place the user was trying to get to)?